site/src/components/radio.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Radio
+description: The radio component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FuVA2amRR71yJZ0GS6RI6zL%2F%25F0%259F%258C%259E-Atlas-Design-Library%3Fnode-id%3D838%253A1096"
+---
+
+# Radio
+
+A radio button allows users to select a single option. It is usually associated with form submissions.
+
+| Type      | Default State                                                                                                                                                | Disabled                                                                                                                                                              |
+| --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Unchecked | <label class="radio" title=""><input type="radio" name="question-0" value="0" class="radio-dot"> <span class="radio-label-text">50%</span></label>           | <label class="radio" title=""><input type="radio" name="question-0" value="1" class="radio-dot" disabled> <span class="radio-label-text">70%</span></label>           |
+| Checked   | <label class="radio" title=""><input type="radio" name="question-0-1" value="0" class="radio-dot" checked> <span class="radio-label-text">50%</span></label> | <label class="radio" title=""><input type="radio" name="question-0-2" value="1" class="radio-dot" checked disabled> <span class="radio-label-text">70%</span></label> |
+
+## Usage
+
+Here is an example of radio buttons. Make sure to use the same value for the `name` attribute within a group of radio buttons.
+
+```html
+<label class="radio">
+	<input checked name="question-1" type="radio" class="radio-dot" value="0" />
```

Comment:
nit: Shouldn't the `value` match the label's text? I know it doesn't matter much in these examples but I think it would be nice to follow the standards. Same for all examples

Reply:
I see, still, I don't think we should rely on assessments here, and better to follow the general html standards in case someone outside the docs world would land on a page and check the examples.

Reply:
updated